src/app/dune

```diff
@@ -106,25 +111,25 @@
             ocplib_i18n)
  (modules Learnocaml_description_main)
  (preprocess (pps ppx_ocplib_i18n js_of_ocaml-ppx))
+ (preprocessor_deps (sandbox none))
 )
 
 
-(executable
- (name learnocaml_partition_view)
- (modes byte js)
- (flags :standard -warn-error -9-27-33)
- (libraries asak
-	    ezjsonm
-            grading_jsoo
-            ace
-            learnocaml_repository
-            learnocaml_app_common
-            lwt_react
-            js_of_ocaml-ppx
-            ocplib_i18n)
- (modules Learnocaml_partition_view)
- (preprocess (pps ppx_ocplib_i18n js_of_ocaml-ppx))
-)
+;; (executable
+;;  (name learnocaml_partition_view)
+;;  (modes byte js)
+;;  (flags :standard -warn-error -9-27-33)
+;;  (libraries ezjsonm
+;;             grading_jsoo
+;;             ace
+;;             learnocaml_repository
+;;             learnocaml_app_common
+;;             lwt_react
+;;             js_of_ocaml-ppx
+;;             ocplib_i18n)
+;;  (modules Learnocaml_partition_view)
+;;  (preprocess (pps ppx_ocplib_i18n js_of_ocaml-ppx))
+;; )
```

Comment:
should/could this stanza be un-commented,
as a follow-up of https://github.com/ocaml-sf/learn-ocaml/pull/602/commits/787440dabbbdfbc31daae6792c85c682ea9a8936 ?

Reply:
Oops, it definitely should indeed. It's actually weird that I could verify the partition view from the web-app worked without this... did I forget to commit something ? I'll check again.